src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/util/OcrDataDeserializer.java

```diff
@@ -41,7 +42,11 @@ public OcrDataDeserializer() {
         return ocrData.getFields().stream().collect(
             toMap(
                 field -> field.getName().textValue(),
-                field -> field.getValue().asText("")
+                field -> field.getValue().asText(""),
+                (u, v) -> {
```

Comment:
what's `u` and `v`?

Reply:
may be key1,key2 ?

Reply:
unfortunately, we lose the information about the duplicate key this way... :(
Perhaps we could identify whether there are any duplicates in the list before we convert it to a map?

Reply:
I've created another jira ticket to do this as this pr is blocking sscs UAT testing